styleguide/components/21-atoms/list/_list.scss

```diff
@@ -43,6 +46,7 @@ ul {
 
   &.iconlist-small {
     li {
+      padding-left: 1.4rem;
```

Comment:
Just set the padding in 'em', that's much more fool proof as it will resize according to whatever font size we use.
(maybe also for line 40, not sure what that's for based on the code)